### ISSUE TITLE
Can't verify if From: is continued Header

### ISSUE THREAD

**Issue by yuakihiro** [lieser/dkim_verifier#218]
4.0.0pre2

from header is a continued header like
```
From:
	=?utf-8?q?(omit)?=
	=?utf-8?q?(omit)?= <mail@example.com>
```
verification is failed
```
DKIM_Verifier.background	FATAL	 Unexpected error during onMessageDisplayed Error: Unexpected start of from header
    parseFromHeader moz-extension://525ad9bc-c7c7-4bc7-9df0-679dc93f1a17/modules/msgParser.mjs.js:142
    verify moz-extension://525ad9bc-c7c7-4bc7-9df0-679dc93f1a17/modules/AuthVerifier.mjs.js:120
background.mjs.js:169:7
```
From:\r\n\t=?utf-8?q?(omit)?=...
code assumes space after From: like "from: "

**Comment by lieser** [comment]
Thanks a lot for reporting. Can reproduce the problem.
Will try to finish a new per-release there this gets fixed this weekend.

**Comment by lieser** [comment]
Should be fixed with version the 4.0.0pre3 that I just released.
